precompiles/outposts/osmosis/osmosis.go

```diff
@@ -42,8 +42,7 @@ type Precompile struct {
 	cmn.Precompile
 	wevmosAddress common.Address
 	// IBC
-	timeoutHeight    clienttypes.Height
-	timeoutTimestamp uint64
+	timeoutHeight clienttypes.Height
```

Comment:
do we need this field? Why not defining a defaultTimeoutHeight var on the ICS20 types and use that. Or create the timeoutHeight with the default values when needed?